pocs/mount/mount.py

```diff
@@ -311,6 +311,106 @@ def distance_from_target(self):
 
         return separation
 
+    def get_tracking_correction(self, offset_info, pointing_ha):
+        """Determine the needed tracking corrections from current position.
+
+        Args:
+            offset_info (`OffsetError`): A named tuple describing the offset
+                error. See `pocs.images.OffsetError`.
+            pointing_ha (float): The Hour Angle (HA) of the mount at the
```

Comment:
Is the point here that the mount will do a meridian flip to go from 89 degrees to 91 degrees (90 being pointing to the meridian)? And thus the HA will tell us if we started observing with the cameras on the left or right side of the pier?

Reply:
I added some more info in the docstring.